config/someip-config.json

```diff
@@ -0,0 +1,83 @@
+{
+    "unicast" : "172.17.0.2",
+    "Streamer":
```

Comment:
Streamer should NOT have an id as no one is suppose to send messages to this address.